src/routes/DisciplineSelectionScreen.tsx

```diff
@@ -27,12 +28,9 @@ const StyledList = styled(FlatList)`
   width: 100%;
 ` as ComponentType as new () => FlatList<Discipline>
 
-const Description = styled.Text<{ selected: boolean }>`
+const Description = styled(ContentSecondaryLight)<{ selected: boolean }>`
   text-align: center;
-  font-size: ${props => props.theme.fonts.defaultFontSize};
-  font-family: ${props => props.theme.fonts.contentFontRegular};
   padding-left: ${props => props.theme.spacings.xxs};
-  font-weight: ${props => props.theme.fonts.lightFontWeight};
   color: ${prop => (prop.selected ? prop.theme.colors.background : prop.theme.colors.textSecondary)};
```

Comment:
i was not sure if its good also to move the `selected` into a component because the colors may differ from case to case. sth like `ContentSecondaryLightSelectable`
I think its getting too specific thats why i spared it out

Reply:
We will get merge conflicts with #240 anyway :D But I think it is really fine to spare it.